docs/walk-through/about-hera.md

```diff
@@ -4,49 +4,27 @@ Hera is a Python library that allows you to construct and submit Argo Workflows.
-> * Architects: 20%
-> * Data Engineer / Data Scientist / ML Engineer: 13%
-
-We hope by providing a more intuitive Python definition language, Data and ML users of Argo Workflows will increase.
-
-## Feature Parity
-
-A natural concern about an abstraction layer on top of another technology is whether it can function the same as the
-original lower layer. In this case, Hera generates a [library of model classes](../api/workflows/models.md) using
-Argo's OpenAPI spec which are wrapped up by Hera's feature-rich classes, while the model classes are available as a
-fallback mechanism. You can check out the extensive
-["upstream" examples](../examples/workflows/upstream/dag_diamond.md) that contain side-by-side Python and YAML
-definitions for Workflows in
-[the Argo examples folder on GitHub](https://github.com/argoproj/argo-workflows/tree/master/examples). Our CI/CD runs
-through the Argo examples folder to check that we are able to reproduce them using Hera Workflows written by hand (note:
-we have not _yet_ written Hera Workflows for all the examples).
-
 If you are a new user of Argo, we encourage you to become familiar with
 [Argo's Core Concepts](https://argoproj.github.io/argo-workflows/workflow-concepts/), which provide a foundation of
 understanding when working with Hera. Working through the
 [Argo Walkthrough](https://argoproj.github.io/argo-workflows/walk-through/) will also help you understand key concepts
 before moving to Python.
 
-## Context Managers
+## Feature Parity
 
-You will notice many classes in Hera implement the context manager interface. This was designed to mirror the YAML
-syntax of Argo, helping existing users come to Hera from YAML, and for users new to both Argo and Hera, who will be able
-to interpret and understand most of the existing YAML documentation and resources online from familiar naming and
-functionality in Hera. Read more in the [user guides](../user-guides/core-concepts.md) section!
+Hera offers hand-written custom classes with features to make Workflow-authoring easier, but also includes
+auto-generated classes from the Argo Workflows OpenAPI specification as a backup, which integrate with the custom
+classes. This allows you to, as mentioned above, write everything in Hera, without the use of YAML.
 
-## Orchestrating Scripts
+You can check out the extensive ["upstream" examples](../examples/workflows/upstream/dag_diamond.md) that contain
+side-by-side Python and YAML definitions for Workflows in
+[the Argo examples folder on GitHub](https://github.com/argoproj/argo-workflows/tree/master/examples). Our CI/CD runs
+through the Argo examples folder to check that we are able to reproduce them by comparing to Workflows written with Hera
+by hand.
 
-A natural extension of a Python DSL for Argo is tighter integration with Python scripts. This is where Hera improves the
-developer experience through its tailored classes and syntactic sugar to enable developers to easily orchestrate Python
```

Comment:
maybe "another place Hera improves"?

Reply:
I removed this part in the spirit of "show don't tell" (i.e. the examples should make the benefits clear), and to accommodate people finding Hera first or at the same time as Argo (in which case they have no prior knowledge of how annoying YAML is).